### PR TITLE
Remove update key from simulation model

### LINKED ISSUE #1016: Dropping update_key_hash from from model
Based on https://github.com/yaptide/yaptide/pull/836. update_key_hash in model removed and replaced by JWT token. Change made for doing less db reads/writes.

### PR DESCRIPTION
**Pull request by jagodek**

### CODE REVIEW

File: pytest.ini [hunk 1]
```diff
@@ -8,12 +8,12 @@ env =
...
     FLASK_DEBUG=True
-    FLASK_SERVER_NAME=localhost
+    FLASK_SERVER_NAME=127.0.0.1
     # default in-memory database for testing
     FLASK_SQLALCHEMY_DATABASE_URI=sqlite://
     # internal address of the flask app
-    BACKEND_INTERNAL_URL=http://localhost:5000
-
+    BACKEND_INTERNAL_URL=http://127.0.0.1:5000
+    # address of redis server
```
Comment (grzanka) [inline pytest.ini]:
```suggestion
```

File: tests/integration/test_list_simulations.py [hunk 2]
```diff
@@ -96,3 +98,4 @@ def test_list_simulations(celery_app,
     assert data["page_count"] == 3
     assert len(data["simulations"]) == 1
     assert data["simulations"][0]["start_time"] == start_time_of_newest_simulation
+    sleep(5)
```
Comment (grzanka) [inline tests/integration/test_list_simulations.py:104]: That cannot be accepted as solution, I know it may fix the issue, but we need to understand the source of the problems

File: yaptide/persistence/models.py [hunk 3]
```diff
@@ -201,7 +191,13 @@ def update_state(self, update_dict: dict):
             self.simulated_primaries = update_dict["simulated_primaries"]
         if "task_state" in update_dict and self.task_state != update_dict["task_state"]:
             self.task_state = update_dict["task_state"]
+            if self.task_state == EntityState.COMPLETED.value:
+                self.simulated_primaries = self.requested_primaries
```
Comment (grzanka) [inline yaptide/persistence/models.py:195]: maybe that change is causing the issues in the tests ?

File: yaptide/routes/common_sim_routes.py [hunk 4]
```diff
@@ -136,11 +132,18 @@ def post():
...
+
+        logging.debug("Marking simulation tasks as completed")
+
+        # If simulation is too short, it can happen that the simulation completes,
+        # but tasks didnt' sent their completion status or simulated_primaries = requested_primaries.
+        # To avoid this, we update simulation tasks by their final status
+        tasks = fetch_tasks_by_sim_id(sim_id=simulation.id)
+        for task in tasks:
+            update_task_state(task, {"task_state": EntityState.COMPLETED.value})
+
```
Comment (grzanka) [inline yaptide/routes/common_sim_routes.py:146]: here another unrelated change

File: yaptide/routes/utils/tokens.py [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
 from typing import Union
-
 import jwt
 
 SECRET_KEY_TOKEN = token_hex(256)
 SECRET_KEY_TOKEN_REFRESH = token_hex(256)
 _Refresh_Token_Expiration_Time = 120  # minutes
 _Access_Token_Expiration_Time = 10  # minutes
 _Keycloak_Token_Expiration_Time = 30  # minutes
+_Simulation_Token_Expiration_time = 7  # days
```
Comment (grzanka) [inline yaptide/routes/utils/tokens.py:11]: why 7 days ?

Comment (jagodek) [reply yaptide/routes/utils/tokens.py:11]: not my decision, just added extra space here.

File: yaptide/persistence/models.py (hunk 3 above)
Comment (grzanka) [reply yaptide/persistence/models.py:195]: can you remove these unreated changes and see if tests fail as well ?
